src/components/Player.vue

```diff
@@ -171,6 +171,15 @@ export default {
         this.setPlaying(false);
       });
     }
+    window.addEventListener("keydown", (e) => {
+      if (e.code === "Space") {
+        if (this.playlistTracks.length > 0) {
```

Comment:
Maybe it's a good idea to put this check in `store/player.js`? This way it also applies to all other events that modify this (like the mediaSession controls above).

Reply:
I've changed `setPlaying` to check the playlist length first before changing the value and added a `togglePlaying` action.